spec/unit/resources/scope/change_resource_assignments_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './../../../spec_helper'
+
+describe 'oneview_test_api300_synergy::scope_change_resource_assignments' do
```

Comment:
It would be good also to have a test case that test if it fails when a resource does not exists (retrieve! returns false)